backend/src/apiserver/client/kfam.go

```diff
@@ -0,0 +1,85 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"github.com/pkg/errors"
+)
+
+type KFAMInterface interface {
```

Comment:
KFAMInterface -> KFAMClientInterface

Reply:
Done.

Reply:
In fact, why not KFAMInterface? I've seen other interface definitions: ScheduledWorkflowInterface, WorkflowInterface. Looks like the coding style is to declare the variable to contain the 'client': swfClient, wfClient. 
WDYT?